### PR TITLE
Some documentation added to strutils

### PR DESCRIPTION
**Pull request by eqperes**
- Added some `runnableExamples`
- Corrected documentation for `abbrev`

### CODE REVIEW

File: lib/pure/strutils.nim [hunk 1]
```diff
@@ -75,39 +75,61 @@ proc isAlphaAscii*(c: char): bool {.noSideEffect, procvar,
...
+    doAssert isLowerAscii('E') == false
+    doAssert isLowerAscii('7') == false
   return c in {'a'..'z'}
 
 proc isUpperAscii*(c: char): bool {.noSideEffect, procvar,
   rtl, extern: "nsuIsUpperAsciiChar".} =
   ## Checks whether or not `c` is an upper case character.
   ##
   ## This checks ASCII characters only.
+  runnableExamples:
```
Comment (skilchen) [inline lib/pure/strutils.nim]: a copy&paste mistake?

Comment (eqperes) [reply lib/pure/strutils.nim]: Yep. I've pushed a fix now.

**Comment by skilchen** [comment]
You can run `nim doc strutils.nim` locally to see if your `runnableExamples` really work.

**Comment by timotheecour** [comment]
@eqperes
how about:
```
doAssert isAlphaAscii('e') == true
```
=>
```
doAssert isAlphaAscii('e')
```

**Comment by eqperes** [comment]
@timotheecour 
I thought about it, but it seemed to me that `== true` was more explicit about the boolean type of the output of the `proc`. But please let me know if it is a bad practice.

**Comment by timotheecour** [comment]
well Nim doesn't have implicit bool conversion, unlike C++ + others, so it's already explicit :-) 
eg:
```
rg1 doAssert | grep '== true'|wc -l
      45
```

and a majority of these are in the `lib/pure/strutils.nim` you introduced :)

**Pull request merged**